packages/horizon/contracts/staking/HorizonStaking.sol

```diff
@@ -544,8 +544,8 @@ contract HorizonStaking is HorizonStakingBase, IHorizonStakingMain {
             emit StakeWithdrawn(serviceProvider, _tokens);
         } else {
             // Before locking more tokens, withdraw any unlocked ones if possible
-            if (sp.__DEPRECATED_tokensLockedUntil != 0 && block.number >= sp.__DEPRECATED_tokensLockedUntil) {
```

Comment:
Can we have `p.__DEPRECATED_tokensLockedUntil = 0` and `sp.__DEPRECATED_tokensLocked != 0`? I think not no?

Reply:
a good question, I assume that's not possible right?